setup.py

```diff
@@ -102,7 +103,7 @@ def init_packages():
 
 class BuildExt(build_ext):
     def get_ext_filename(self, ext_name):
-        return f"{os.path.join(*ext_name.split('.'))}.so"
+        return f"{platform.uname().machine}/{ext_name}.so"
```

Comment:
Pretty hard to review this without having a bunch of machines on my hand. Can you show an example of what this used to return, and what it does now?